core/src/main/resources/filodb-defaults.conf

```diff
@@ -70,6 +70,12 @@
 
       # This is to start flush on memtable when there is no write activity
       noactivity.flush.interval = 10 s
+
+      # Location to store memtable write ahead log files for each dataset
+      memtable-wal-dir = /tmp/filodb/wal
+
+      # Minimum size of the Memory mapped ByteBuffer allocated for write ahead log file in bytes
+      mapped-byte-buffer-size = 1024
```

Comment:
Sure, we can keep it more. What should be a recommended size, can it be 64k or 1 MB?